README.md

```diff
@@ -88,17 +88,18 @@ requires and provides, the permissions, and the additional module metadata.
-| KAFKA_PORT  |     9092      | Kafka broker port                                                                                                                                      |
-| ENV         |     folio     | Logical name of the deployment, must be set if Kafka/Elasticsearch are shared for environments, `a-z (any case)`, `0-9`, `-`, `_` symbols only allowed |
+| Name                 |     Default value     | Description                                                                                                                                                 |
+|:---------------------|:---------------------:|:------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| DB_HOST              |       postgres        | Postgres hostname                                                                                                                                           |
+| DB_PORT              |         5432          | Postgres port                                                                                                                                               |
+| DB_USERNAME          |      folio_admin      | Postgres username                                                                                                                                           |
+| DB_PASSWORD          |           -           | Postgres username password                                                                                                                                  |
+| DB_DATABASE          |     okapi_modules     | Postgres database name                                                                                                                                      |
+| KAFKA_HOST           |         kafka         | Kafka broker hostname                                                                                                                                       |
+| KAFKA_PORT           |         9092          | Kafka broker port                                                                                                                                           |
+| ENV                  |         folio         | Logical name of the deployment, must be set if Kafka/Elasticsearch are shared for environments, `a-z (any case)`, `0-9`, `-`, `_` symbols only allowed      |
+| SYSTEM_USER_NAME     | consortia-system-user | Username of the system user                                                                                                                                 |
+| SYSTEM_USER_PASSWORD |           -           | Password of the system user                                                                                                                                 |
```

Comment:
I thought that this code will throw an IllegalArgumentException and we don't need to add custom validation in application main(), but not, because it checking this placeholder ${SYSTEM_USER_PASSWORD} and not exact value of it
![image](https://github.com/folio-org/mod-consortia/assets/25097693/dd3f14ba-df90-4364-8e0f-958452ddc6eb)